_includes/people/wilson_greg.html

```diff
@@ -1,6 +1,6 @@
 <p id="wilson_g" data-toggle="tooltip" title="<img src='{{site.filesurl}}/people/wilson_g.jpg' alt='Greg Wilson' />">
 <span class="person"><i>Greg Wilson</i></span>
 co-founded the Software Carpentry project in 1998 after receiving
-his PhD in Computer Science from the University of Edinburgh. He is
-currently the Director of Instructor Training for the Software Carpentry Foundation.
+his PhD in Computer Science from the University of Edinburgh. He recently served as 
+  the Director of Instructor Training for the Software Carpentry Foundation.
```

Comment:
how about? "He was previously the Executive Director and Director of Instructor Training for the Software Carpentry Foundation and now works at Data Camp."